tests/components/mqtt/test_discovery.py

```diff
@@ -270,6 +270,7 @@
     # Undocumented device configuration
     "CONF_DEPRECATED_VIA_HUB",
     "CONF_VIA_DEVICE",
+    "CONF_COMMANDS",
```

Comment:
You can't add an entry to undocumented… document it instead?

Reply:
So, as long as it is documented, CONF_COMMANDS is allowed to be in this whitelist? I made a comment about this in this PR but didn't get a response so I was unsure what to do. I have a documentation PR on home-assistant.io that documents the commands configuration.